lib/cfonb/operation.rb

```diff
@@ -2,6 +2,10 @@
 
 module CFONB
   class Operation
+    class AlreadyDefinedOperationError < StandardError; end
+    class UnstartedOperationError < StandardError; end
+    class UnhandledLineCodeError < StandardError; end
```

Comment:
Please move errors to `lib/cfonb/error.rb` so that they are all defined in one place.